Assignment6.Rmd

```diff
@@ -66,20 +68,58 @@ M2$predict1 <- predict(c.tree1, M2, type = "class")
 
 M2$predict2 <- predict(c.tree2, M2, type = "class")
 
-table(M2$certified, M2$predict1)
+T1 <- table(M2$certified, M2$predict1)
+T1
+T2 <- table(M2$certified, M2$predict2)
+T2
+```
+### Model 1
 
-table(M2$certified, M2$predict2)
+```{r}
+# true positive
+T1[2,2]/sum(T1)
 
+# true negative
+T1[1,1]/sum(T1)
 ```
+### Model 2
+```{r}
+# true positive
+T2[2,2]/sum(T2)
 
-##Part III
+# true negative
+T2[1,1]/sum(T2)
+```
+
+### explanation
+Second model, second tree has greater true positive and overall accuracy. It shows that the pruned model is more generaliable in the data. 
```

Comment:
Can you please specify the exact accuracy rate of the model in your explanation?